pinot-core/src/test/resources/data/fakestream/fake_stream_pinot_schema.json

```diff
@@ -0,0 +1,335 @@
+{
+  "schemaName": "mytable",
```

Comment:
Do we need to checkin the schema? Can we generate it from the avro file given? (since fake stream always reads from avro file)

Reply:
I wanted to avoid extracting the data once more. I have no preference either way. It was just easier this way

Reply:
OK, then please add a comment in the schema file that it must be in sync with the avro file of specific path.